scripts/run_test.bat

```diff
@@ -15,26 +15,6 @@ rem See the License for the specific language governing permissions and
 rem limitations under the License.
 rem ============================================================================
 
-set DAAL4PY_VERSION=%PKG_VERSION%
-set MPIROOT=%PREFIX%\Library
-
-IF DEFINED DPCPPROOT (
-    echo "Sourcing DPCPPROOT"
-    call "%DPCPPROOT%\env\vars.bat"
```

Comment:
DPC++ RT is not needed for testing?

Reply:
DPC++ RT sources by the compiler

Reply:
But you removed the line that sources the compiler environment in this change